src/MatrixAlgebraKit.jl

```diff
@@ -30,6 +30,9 @@ export LAPACK_HouseholderQR, LAPACK_HouseholderLQ,
        LAPACK_DivideAndConquer, LAPACK_Jacobi
 export truncrank, trunctol, truncabove, TruncationKeepSorted, TruncationKeepFiltered
 
+VERSION >= v"1.11.0-DEV.469" &&
```

Comment:
Is it worth specifying the specific DEV version that introduced the `public` keyword, or can we just use `v"1.11"`.

Reply:
I just copied that from the Julia docs: https://docs.julialang.org/en/v1.12-dev/manual/modules/#Export-lists.

I also know Compat.jl uses this style of being more specific about the version. I suppose if you just use `v"1.11"` someone could use the package with an earlier dev version of v1.11 before `public` was introduced, obviously that is quite a corner case and it is reasonable to tell users to not do that/that isn't supported.

Reply:
I guess "worth" is subjective, it is a few extra characters and very slightly safer, and something that we write once and never think about again (until we drop support for v1.11 in which case we delete the entire logic anyway).